src/components/Common/Button.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+
+const Button = styled.button`
+  border: 0;
+  display: block;
+  padding: ${remcalc(18)} ${remcalc(24)};
+  color: ${props => props.theme.colors.white};
+  background: ${props => props.theme.colors.text};
+  font-weight: bold;
+  font-size: ${remcalc(18)};
+  line-height: ${remcalc(24)};
+  box-sizing: border-box;
+
+  :disabled {
```

Comment:
does this work? Shouldn't it be `&:disabled` ?

Reply:
No idea. I was just moving the code into a different file :wink:

Reply:
just tried and it works 🤷‍♂️  leave it as is 😄